backend/apps/interest/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+from .models import Interest
+
+
+class InterestSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Interest
+        fields = ["id", "project", "message", "contact_opt_in", "status"]
+        read_only_fields = ["id", "status"]
```

Comment:
In the Interest model there is:

    investor = models.ForeignKey(User)

But in the serializer:

    fields = ["id", "project", "message", "contact_opt_in", "status"]

There is no investor field.

So how does DRF know who creates the interest?

This is usually done in the ViewSet